Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -0,0 +1,143 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import BraveUI
+import DesignSystem
+import BraveShields
+import BraveVPN
+import Data
+
+struct SubmitReportView: View {
+  @Environment(\.dismiss) private var dismiss: DismissAction
+  let url: URL
+  let isPrivateBrowsing: Bool
+  
+  @State private var additionalDetails = ""
+  @State private var contactDetails = ""
+  @State private var isSubmittingReport = false
+  @State private var isSubmitted = false
+  
+  private var scrollContent: some View {
+    ScrollView {
+      VStack(alignment: .leading, spacing: 16) {
+        Text(Strings.Shields.reportBrokenSiteBody1)
+        Text(url.absoluteString)
+          .foregroundStyle(Color(braveSystemName: .textInteractive))
+        Text(Strings.Shields.reportBrokenSiteBody2)
+          .font(.footnote)
+        BraveTextEditor(
+          text: $additionalDetails,
+          prompt: Strings.Shields.reportBrokenAdditionalDetails
+        ).frame(height: 80)
+        
+        VStack(alignment: .leading, spacing: 4) {
+          Text(Strings.Shields.reportBrokenContactMe).font(.caption)
+          TextField(
+            Strings.Shields.reportBrokenContactMe,
+            text: $contactDetails, prompt: Text(Strings.Shields.reportBrokenContactMeSuggestions)
+          )
+          .textContentType(.emailAddress)
+          .textFieldStyle(BraveTextFieldStyle())
+          .autocorrectionDisabled()
+          .textInputAutocapitalization(.never)
+        }
+      }
+      .padding()
+      .foregroundStyle(Color(braveSystemName: .textSecondary))
+    }
+    .background(Color(.braveBackground))
+    .foregroundStyle(Color(braveSystemName: .textSecondary))
+    .navigationTitle(Strings.Shields.reportABrokenSite)
+    .toolbar {
+      ToolbarItem(placement: .cancellationAction) {
+        Button(Strings.cancelButtonTitle) {
+          dismiss()
+        }
+        .disabled(isSubmittingReport)
+        .hidden(isHidden: isSubmitted)
+        .animation(.smooth.speed(0.2), value: isSubmitted)
```

Comment:
`smooth` might be Xcode 15+ IIRC, plus don't alter the speed of an animation like that. In this case `smooth` takes arguments to adjust the speed

Reply:
I just removed the animation, honestly the difference is barely noticeable. I was just trying to get the cancel to gracefully disappear.